templates/helm/templates/cluster-role-binding.yaml.tpl

```diff
@@ -1,11 +1,22 @@
 apiVersion: rbac.authorization.k8s.io/v1
+{{ "{{ if not .Values.namespacedInstallation }}" }}
 kind: ClusterRoleBinding
 metadata:
   name: {{ "{{ include \"app.fullname\" . }}" }}
 roleRef:
   apiGroup: rbac.authorization.k8s.io
   kind: ClusterRole
   name: ack-{{ .ServiceIDClean }}-controller
+{{ "{{ else }}" }}
+kind: RoleBinding
+metadata:
+  name: {{ "{{ include \"app.fullname\" . }}" }}
+  namespace: {{ "{{ .Release.Namespace }}" }}
```

Comment:
agreed

Reply:
The Role needs to be installed into the same namespace as the SA it will be attached to. That SA, and the controller deployment, are deployed into `{{ .Release.Namespace }}`. `.Values.watchNamespace` is an independent variable passed to the controller.

Reply:
> The Role needs to be installed into the same namespace as the SA it will be attached to. That SA, and the controller deployment, are deployed into `{{ .Release.Namespace }}`. `.Values.watchNamespace` is an independent variable passed to the controller.

In the case of a namespaced installation, though, shouldn't they be the same thing? It's not going to work if the namespaces are different...

Reply:
If `{{ .Values.namespacedInstallation }}`, then yes they will need to be the same thing. But we cannot impose a limit on `{{ .Release.Namespace }}`, as that's defined at install-time.